### PR TITLE
bluetooth: tester: MICP & AICS BTP events

### PR DESCRIPTION
**Pull request by piotrnarajowski**
Enhance MICP & AICS BTP events functionality for invalid behavior tests.

### CODE REVIEW

File: tests/bluetooth/tester/src/btp_aics.c [hunk 1]
```diff
@@ -539,17 +566,84 @@ static void aics_description_cb(struct bt_aics *inst, int err, char *description
...
-	btp_send_aics_description_ev(conn, data_len, description);
+	btp_send_aics_description_ev(conn, err, data_len, description);
 
 	LOG_DBG("AICS description callback (%d)", err);
 }
 
+static void aics_set_gain_cb(struct bt_aics *inst, int err)
+{
+	struct bt_conn *conn;
+	uint8_t opcode = 0x02;
```
Comment (Thalley) [inline tests/bluetooth/tester/src/btp_aics.c:577]: Aren't these opcode `#define`d?

Comment (piotrnarajowski) [reply tests/bluetooth/tester/src/btp_aics.c:577]: Right, don't know why I missed that...

File: tests/bluetooth/tester/src/btp_aics.c [hunk 2]
```diff
@@ -539,17 +566,84 @@ static void aics_description_cb(struct bt_aics *inst, int err, char *description
...
+}
+
+static void aics_set_auto_gain_cb(struct bt_aics *inst, int err)
+{
+	struct bt_conn *conn;
+	uint8_t opcode = BTP_AICS_AUTO_GAIN_SET;
+
+	bt_aics_client_conn_get(inst, &conn);
+
+	btp_send_aics_procedure_ev(conn, err, opcode);
```
Comment (Thalley) [inline tests/bluetooth/tester/src/btp_aics.c:629]:
```suggestion
	bt_aics_client_conn_get(inst, &conn);

	btp_send_aics_procedure_ev(conn, err, BTP_AICS_AUTO_GAIN_SET);
```

These could be a bit more simple, but they are also OK as is

**Review by Thalley — DISMISSED** [review]

File: tests/bluetooth/tester/src/btp_aics.c (hunk 2 above)
Comment (piotrnarajowski) [reply tests/bluetooth/tester/src/btp_aics.c]: Fixed

**Review by Thalley — APPROVED** [review]

**Review by sjanc — APPROVED** [review]

**Pull request merged**